ajax/libs/Bounce/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bounce",
+  "filename": "bounce.min.js"
+  "version": "0.0.0",
+  "description": "Bounce.js is a tool and JS library for generating beautiful CSS3 powered keyframe animations.",
+  "homepage": "http://bouncejs.com",
+  "keywords": [
+    "bounce",
+    "bounce.js",
+    "bouncejs.com"
```

Comment:
Let's add the keywords as: `CSS3`, `animations`, `keyframe`, `scale`, `rotate`, `translate`, `skew`, `bounce`.
These would help people find this lib easier.
Thanks